src/components/NewCommentForm.tsx

```diff
@@ -45,24 +96,36 @@ export const NewCommentForm: React.FC = () => {
             name="email"
             id="comment-author-email"
             placeholder="email@test.com"
-            className="input is-danger"
+            className={classNames('input', {
+              'is-danger': !isEmailPresent,
```

Comment:
looks like we can just check
```suggestion
              'is-danger': !!email,
```